data/json/items/comestibles.json

```diff
@@ -8483,7 +8606,7 @@
         "id" : "syrup",
         "name" : "maple syrup",
         "name_plural" : "maple syrup",
-        "weight" : 28,
+        "weight" : 334,
```

Comment:
You sure about that? Did you check how much it actually weighs now at 7 charges per volume?